test/controllers/api/v2/locations_controller_test.rb

```diff
@@ -335,4 +335,63 @@ def add_locations
     assert_response :success
     assert_equal 1, location.reload.location_parameters.count
   end
+
+  test "should create with comma separated name" do
+    name = "#{RFauxFactory.gen_alpha}, #{RFauxFactory.gen_alpha}"
+    post :create, params: { :location => { :name => name } }
+    assert_response :success, "creation with name #{name} failed with code #{response.code}"
+    show_response = JSON.parse(@response.body)
+    refute_empty show_response
+  end
+
+  test "should create with description" do
+    name = RFauxFactory.gen_alpha
+    description = RFauxFactory.gen_alpha
+    post :create, params: { :location => { :name => name, :description => description } }
+    assert_response :success, "creation with name #{name} and description #{description} failed with code #{response.code}"
+    show_response = JSON.parse(@response.body)
+    refute_empty show_response
```

Comment:
if you parse the response on line above, it might be a good idea to check its content, e.g. that `assert_equal description, show_response["description"]` (not sure if it's not nested under location key too)